### PR TITLE
Move Request Items

### PR DESCRIPTION
**Pull request by or4nga**
Added documentation on new `move/requests/items` endpoint

### CODE REVIEW

File: api/endpoints/move.md [hunk 1]
```diff
@@ -304,13 +304,10 @@ A single move reservation object.
     [Move][]
 
 
-## Edit a Move Request [PUT /move/requests///{id}]
```
Comment (GimmeYerGold) [inline api/endpoints/move.md:307]: So while it is possible to update a request without including the ID in the PUT call url, the ID will still be required in the body of the call. Here, both are removed. 

Are move items replace move requests?

**Review by GimmeYerGold — CHANGES_REQUESTED** [review]
Hi Patrick! Just need a little more context on the changes made here please. Thanks!

File: api/endpoints/move.md (hunk 1 above)
Comment (or4nga) [reply api/endpoints/move.md:307]: I believe `parameters` is equivalent to the URL? So while i'm removing both, they are both referring to the same thing.

Move items make up move requests. The structure is like this: 
```
request: {
  id: 1
  items: [
    item1,
    item2,
    ...
  ]
}
```

Comment (or4nga) [reply api/endpoints/move.md:307]: This change is to correct an error in the docs. If you want to edit a move request, you make a PUT request without the ID in the URL, but in the body, so I'm correcting the docs.

Comment (or4nga) [reply api/endpoints/move.md:307]: The main scope of this PR is to add documentation for the `requests/items` endpoint which is a new endpoint that lets you add/edit/delete move request items

Comment (GimmeYerGold) [reply api/endpoints/move.md:307]: ok, thanks! I built the branch and see that ID is still in the body of that call, so we're good. All the rest looks good. 👍

**Review by GimmeYerGold — APPROVED** [review]

**Pull request merged**